api/test/deals.go

```diff
@@ -34,18 +31,7 @@ import (
 	ipld "github.com/ipfs/go-ipld-format"
 )
 
-var MineNext = miner.MineReq{
-	InjectNulls: 0,
-	Done:        func(bool, abi.ChainEpoch, error) {},
-}
-
-func init() {
-	logging.SetAllLoggers(logging.LevelInfo)
-	build.InsecurePoStValidation = true
-}
-
 func TestDealFlow(t *testing.T, b APIBuilder, blocktime time.Duration, carExport, fastRet bool) {
-	_ = os.Setenv("BELLMAN_NO_GPU", "1")
```

Comment:
why remove all this?

Reply:
oh, i see refactored it into init

Reply:
I'm setting it once in an `init` function. Setting globals in different places was making it difficult to tell which globals applied to which tests.

Reply:
Merged in https://github.com/filecoin-project/lotus/pull/4538 (broken out of this PR).